src/main/java/com/redhat/management/approval/EmailBody.java

```diff
@@ -178,7 +178,7 @@ private String getParamsTable(HashMap<String, String> params) {
                 "<table><tbody><tr><td><strong>Key</strong></td><td><strong>Value<strong></td></tr>\n");
         
         for (HashMap.Entry<String, String> entry: params.entrySet()) {
-            String param = "<tr><td>" + entry.getKey() + "</td><td>" + entry.getValue() + "</td></tr>\n";
+            String param = "<tr><td>" + entry.getKey() + "</td><td>" + entry.getValue().toString() + "</td></tr>\n";
```

Comment:
I don't think this solves the problem. Your entry is of type `Entry<String, String>` whose value is already a string. You need to change params to type `Entry<String, Object>`